src/scripts/data-migration.js

```diff
@@ -0,0 +1,26 @@
+class DataMigrationService {
+  constructor() {
+    this.database = new SQLiteDatabaseService();
+    this.oldDatabase = new WebSQLDBService();
+    this.oldDatabase.initializeDB();
```

Comment:
note that we need to delete this logic before chrome 123 else `openDatabase` will begin to throw exceptions. Same goes for the other repo